lib/util/index.js

```diff
@@ -109,15 +109,21 @@ function flatMap(list, mapFn) {
  * @returns {?string} browser - "chrome", "firefox", "safari", or null
  */
 function guessBrowser() {
-  if (typeof webkitRTCPeerConnection !== 'undefined') {
+  if (typeof window === 'undefined' || !window.navigator) {
+    return null;
+  }
+  const navigator = window && window.navigator;
```

Comment:
twilio-webrtc.js is still ES5-only, so please use `var` instead of `const`. Also, because you already checked `window` above, this can just be

```js
var navigator = window.navigator.
```